users/models.py

```diff
@@ -171,3 +173,10 @@ def email_user(self, subject, message, from_email=None):
         Sends an email to this user.
         """
         send_mail(subject, message, from_email, [self.email])
+
+
+@receiver(post_save, sender=CustomUser)
+def update_from_manager_to_employee(sender, **kwargs):
+    user = kwargs['instance']
+    if user.user_type is not CustomUser.UserType.MANAGER.name and user.user_type is not CustomUser.UserType.ADMIN.name:
```

Comment:
This if-statement it much too long. Please split them into at least 2 separate statements.

Reply:
Done